libraries/AdaptiveExpressions/BuiltinFunctions/ComparisonEvaluator.cs

```diff
@@ -34,38 +34,16 @@ private static EvaluateExpressionDelegate Evaluator(Func<IReadOnlyList<object>,
                 (args, error) = FunctionUtils.EvaluateChildren(expression, state, new Options(options) { NullSubstitution = null }, verify);
```

Comment:
`ComparisonEvaluator.cs` seems  do nothing. Just the same as `applyWithError`. Consider remove it?

Reply:
There is swallow error operation in Evaluator of ComparisonEvaluator.cs.
Can't be transformed to ApplyWithError